src/server.ts

```diff
@@ -18,9 +19,16 @@ const build = (opts: FastifyServerOptions): FastifyInstance => {
   fastify.register(multipart);
   fastify.register(FastifySSEPlugin);
   fastify.register(cookie);
+  fastify.register(fastifyCompress);
   fastify.register(view, {
     engine: {
-      liquid: new Liquid({ extname: '.html', root: 'src/liquid', jekyllInclude: true, dynamicPartials: true }),
+      liquid: new Liquid({ 
+        extname: '.html', 
+        root: 'src/liquid', 
+        cache: process.env.NODE_ENV === 'production', 
```

Comment:
The performance tweaks in this PR are spot-on.

For generation times, measured the change to enable `cache: true` in LiquidJS and it brought down time-to-generate by 80% (2.5s down to 0.5s).  
For response size, compression brought responses sizes down 97% (from 3.5MB to ~125kb).